From 59077f39cce499d11de5b0a9c23c9ded67ddb22e Mon Sep 17 00:00:00 2001
From: Jeremy Puhlman <jpuhlman@mvista.com>
Date: Wed, 11 Mar 2020 19:56:55 +0000
Subject: [PATCH] strace: fix reproducibilty issues

The full path to the gen_tests.sh script is encoded in the tests

Upstream-Status: Pending

Signed-off-by: Jeremy Puhlman <jpuhlman@mvista.com>
---
 tests/gen_tests.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/gen_tests.sh b/tests/gen_tests.sh
index 3c6228a..43b94ea 100755
--- a/tests/gen_tests.sh
+++ b/tests/gen_tests.sh
@@ -46,7 +46,7 @@ while read -r name arg0 args; do {
 
 	hdr="\
 #!/bin/sh -efu
-# Generated by $0 from $input ($name $arg0 $args); do not edit."
+# Generated by $(basename $0) from $input ($name $arg0 $args); do not edit."
 
 	case "$arg0" in
 		+*)
@@ -102,7 +102,7 @@ while read -r name arg0 args; do {
 
 if [ -n "$names" ]; then
 	{
-		printf '# Generated by %s from %s; do not edit.\n' "$0" "$input"
+		printf '# Generated by %s from %s; do not edit.\n' "$(basename $0)" "$input"
 		printf 'GEN_TESTS ='
 		printf ' %s.gen.test' $names
 		echo
